addon/components/hyper-table-v2/manage-columns.hbs

```diff
@@ -29,19 +29,12 @@
     </div>
     <div class="available-fields-wrapper__fields">
       <div class="search">
-        <OSS::InputContainer>
-          <:prefix>
-            <OSS::Icon @icon="fa-search" />
-          </:prefix>
-
-          <:input>
-            <Input
-              @value={{this._searchColumnDefinitionKeyword}}
-              placeholder={{t "hypertable.features.manage_fields.search_placeholder"}}
-              data-control-name="column_definitions_search_input"
-            />
-          </:input>
-        </OSS::InputContainer>
+        <OSS::SearchField
+          data-control-name="column_definitions_search_input"
```

Comment:
NIt: same comment as below, usually `data-control-name`s are defined under the other attributes